index.js

```diff
@@ -22,14 +21,22 @@ const exec = (cmd, args) => {
 	).filter(Boolean);
 };
 
+const install = (bin, useYarn) => () => execa(bin, ['install'])
```

Comment:
Thanks for the comprehensive reply Sam. I too think `2` would be the best solution right now, but would like to see `3` be possible eventually.

---

> Add something like a visible or disabled property to Listr to dynamically show and hide tasks. That way we could use the skip function to detect if yarn is installed.

Sounds useful, but why not also make `skip` dynamic? You're pretty much treating the terminal as a canvas anyways, so anything could potentially be dynamic.

@SamVerschueren Could you open an issue about this on Listr, so we could continue this discussion there?